### PR TITLE
MEN-4258: login: ask for the user password when not provided with "--password" arg

### PR DESCRIPTION
**Pull request by kjaskiewiczz**
Issue: https://tracker.mender.io/browse/MEN-4258

### CODE REVIEW

File: cmd/login.go [hunk 1]
```diff
@@ -93,9 +93,6 @@ func NewLoginCmd(cmd *cobra.Command, args []string) (*LoginCmd, error) {
 	}
 
 	password := viper.GetString(argLoginPassword)
-	if password == "" {
-		return nil, errors.New("No password set")
-	}
```
Comment (merlin-northern) [inline cmd/login.go:98]: looks good to me. on master, the same if with the `username` is not present as well.

**Review by merlin-northern — APPROVED** [review]

File: cmd/login.go (hunk 1 above)
Comment (kjaskiewiczz) [reply cmd/login.go:98]: it's different because on master we also prompt for username, but in 1.5.x we don't

Comment (oleorhagen) [reply cmd/login.go:98]: @kjaskiewiczz could you cherry this instead: https://github.com/mendersoftware/mender-cli/commit/a03979a283e04af4cbbebfb81a2e197f41cca02c ?

Comment (lluiscampos) [reply cmd/login.go:98]: @oleorhagen  That can be cherry-picked (or we can reopen your PR) but requires a `Changelog` tag

**Comment by oleorhagen** [comment]
Follow up cherry: https://github.com/mendersoftware/mender-cli/pull/94

**Pull request closed without merging**